.changes/2318-sync-fixes.md

```diff
@@ -0,0 +1 @@
+- Fix to not refresh the entire space history on every app start anymore.
```

Comment:
So now, it'd only sync if there's a need for sync ?

Reply:
yeah, that's the idea. so far, we synced the entire history on every start...